src/core/logger.cpp

```diff
@@ -65,7 +65,7 @@ void Logger::log(LogLevel level, const Class *class_, const char *file,
     }
 
     std::string text = d->formatter->format(level, class_,
-        Thread::thread(), file, line, msg);
+        file, line, msg);
 
     std::lock_guard<std::mutex> guard(d->mutex);
```

Comment:
I didn't realize this was already guarded by a mutex, so actually there's nothing new to do now that the logger is process-wide by default.